ArticleTemplates/assets/scss/modules/content/_standfirst.scss

```diff
@@ -24,4 +24,9 @@
     b {
         font-weight: 700;
     }
+
+    body:not(.garnett--type-analysis):not(.garnett--type-live) & {
```

Comment:
Is it possible to have a comment article that isn't `analysis`?

Reply:
Does `body` work? I thought that was targeting the `body` element which is outside of `.standfirst`?

Reply:
What I'm trying to do is to exclude analysis (the articles with the underlined headlines like [this](https://www.theguardian.com/politics/2020/may/26/inconsistencies-between-cummings-lockdown-story-and-his-wifes)) and liveblogs.

Targetting `body` t works with the `&` after — so that applies to `body:not(.garnett--type-analysis):not(.garnett--type-live) .standfirst`!